### ISSUE TITLE
v16 - Sending focused window to left or right space on secondary screen will move the window to the main display

### ISSUE THREAD

**Issue by juzuze** [ianyh/Amethyst#1299]
**Describe the bug**
when using multiple monitors when I try and send the focused window to the left / right space it instead sends the focused window to the main display, then the space moves to the right / left

### expected result
- focused window moves to the left / right space relative to its current focused space

### actual result
- focused window is sent to primary monitor and space is shifted to the right / left (doesn't take the focused window with it) 

**To Reproduce**
- on a second screen use the keyboard shortcut to sent to right / left space

**Versions:**
 - macOS: 12.5.1 (M1)
 - Amethyst: v16.0
 
**Debug Info**
```
Version: 0.16.0 (94)

OS version: Version 12.5.1 (Build 21G83)

Screens:
	(0.0, 0.0, 3440.0, 1440.0) [(0.0, 0.0, 3440.0, 1440.0)]
	(-1280.0, 0.0, 1280.0, 800.0) [(-1280.0, 640.0, 1280.0, 800.0)]

Configuration:
window-margin-size: 4
ignore-menu-bar: 0
floating: (
        {
        id = "org.openvpn.client.app";
        "window-titles" =         (
        );
    },
        {
        id = "com.apple.archiveutility";
        "window-titles" =         (
        );
    },
        {
        id = "com.authy.authy-mac";
        "window-titles" =         (
        );
    },
        {
        id = "com.apple.Home";
        "window-titles" =         (
        );
    },
        {
        id = "com.apple.Stickies";
        "window-titles" =         (
        );
    }
)
focus-follows-mouse: 0
mod1: (
    option,
    shift
)
layouts: (
    tall,
    wide,
    fullscreen,
    "two-pane",
    floating,
    "middle-wide"
)
screen-padding-left: 0
use-canary-build: 1
floating-is-blacklist: 1
window-minimum-width: 0
mouse-swaps-windows: 0
mouse-follows-focus: 1
mouse-resizes-windows: 1
follow-space-thrown-windows: 1
float-small-windows: 1
screen-padding-right: 0
window-resize-step: 5
window-minimum-height: 0
enables-layout-hud: 1
window-margins: 1
mod2: (
    option,
    shift,
    control
)
debug-layout-info: 1
screen-padding-bottom: 0
restore-layouts-on-launch: 0
smart-window-margins: 1
screen-padding-top: 0
enables-layout-hud-on-space-change: 0
new-windows-to-main: 0
```
Note: `--include-apps` will list your manageable applications, but is optional if you don't want to list that.

**Comment by JordyScript** [comment]
I'm having the same issue.

**Comment by OneHoopyFrood** [comment]
Unchecking Mission Control -> Displays have separate Spaces and relogging is a workaround, though not ideal. I like having separate spaces for each display.

**Comment by ianyh** [comment]
Addressed in #1408
